packages/expo-image/src/Image.tsx

```diff
@@ -79,6 +80,17 @@ export default class Image extends React.Component<ImageProps, ImageState> {
     return await ExpoImageModule.prefetch(url);
   }
 
+  /**
+   * Resolves an asset reference into an object which has the properties `uri`, `width` and `height`
+   *
+   * @param source A number (opaque type returned by require('./foo.png')) or an `ImageSource`.
+   *
+   * @return an object constaining `uri` `width` and `height`.
+   */
+  static resolveAssetSource(source: ImageSourcePropType): object {
```

Comment:
```suggestion
  static resolveAssetSource(source: ImageSourcePropType): ImageAssetSource {
```
Please define somehow the return type, RN image defines it as follows: _(copied from `react-native/index.d.ts`)_
```ts
/**
 * @see https://reactnative.dev/docs/image#resolveassetsource
 */
export interface ImageResolvedAssetSource {
    height: number;
    width: number;
    scale: number;
    uri: string;
}
```